docs/data/base/components/slider/slider.md

```diff
@@ -1,193 +1,214 @@
 
-## Customization
+```tsx
+<Slider.Root step={4} defaultValue={3}>
+  <Slider.Output />
+  <Slider.Track>
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-### Vertical
+You can specify the `largeStep` prop to change the step when the user holds the <kbd>shift</kbd> key, snapping to multiples of 10 by default.
 
-Slider components can be arranged vertically as well as horizontally.
+## Range Sliders
 
-When vertical, you must set `orientation="vertical"` on the Slider so the user can navigate with the up and down arrow keys (rather than the default left-to-right behavior for horizontal sliders).
+To let users set the start and end of a range on a Slider, provide an array of values to the `value` or `defaultValue` prop, and place a `<Slider.Thumb />` for each value in the array:
 
-{{"demo": "VerticalSlider.js"}}
+```tsx
+<Slider.Root defaultValue={[45, 70]}>
+  <Slider.Output />
+  <Slider.Track>
+    <Slider.Thumb />
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-### Discrete sliders
+{{"demo": "RangeSlider.js"}}
 
-The most basic Slider is _continuous_, which means it does not have pre-defined (_discrete_) values for the user to select from.
-This is suitable for situations in which an approximate value is good enough for the user, such as brightness or volume.
+### Overlapping values
 
-But if your users need more precise options, you can create a discrete Slider that snaps the thumb to pre-defined stops along the bar.
-Make sure to adjust the `shiftStep` prop (the granularity with which the slider can step when using Page Up/Down or Shift + Arrow Up/Down) to a value divadable with the `step`.
+The `minDifferenceBetweenValues` prop can be used to to set the mininum difference between values in a range slider, so thumbs do not overlap in the same position. In the below example, the thumbs cannot be moved further towards each other as the difference between values would exceed `minDifferenceBetweenValues`:
 
-To generate a mark for each stop, use `marks={true}`:
+```tsx
+<Slider.Root minDifferenceBetweenValues={5} defaultValue={[15, 20]}>
+  <Slider.Track>
+    <Slider.Thumb />
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-{{"demo": "DiscreteSlider.js"}}
+## Vertical
 
-#### Custom marks
+To create vertical sliders, set the `orientation` prop to `"vertical"`. This will track thumb movements vertically instead of horizontally.
 
-You can create custom marks by providing a rich array to the `marks` prop:
+```jsx
+<Slider.Root orientation="vertical">{/* Subcomponents */}</Slider.Root>
+```
 
-{{"demo": "DiscreteSliderMarks.js"}}
+{{"demo": "VerticalSlider.js"}}
 
-#### Restricted values
+## RTL
 
-If the user should only be able to select from the values provided with the `marks` prop, add `step={null}` to disable all other options:
+Use the `isRtl` prop to set the slider's direction to RTL:
```

Comment:
Tabs use `direction: 'ltr' | 'rtl'`. Could you make it consistent?

Reply:
Updated